### ISSUE TITLE
Option to not clear the screen when in watch mode?

### ISSUE THREAD

**Issue by NullVoxPopuli** [typed-ember/glint#597]
Over in the v2-addon blueprint, we're proposing changing compilation to use:
```
    "start": "concurrently 'npm:start:*'",
    "start:js": "rollup --config --watch --no-watch.clearScreen",
    "start:types": "glint --build --watch",
```
https://github.com/embroider-build/addon-blueprint/pull/136

But, it seems `glint` doesn't have a flag to disable the screen clearing? 
I see that this is the file where the option would be added: https://github.com/typed-ember/glint/blob/main/packages/core/src/cli/index.ts, but how does screen clearing work? https://github.com/search?q=repo%3Atyped-ember%2Fglint%20clear&type=code

**Comment by dfreeman** [comment]
> how does screen clearing work

That's not something we're doing ourselves, it's TypeScript. They have a `--preserveWatchOutput` flag, though, and that may be something we can pass through to the compiler API—but some of their CLI flags don't have any public programmatic equivalent, so someone will have to do a bit of investigation there.

**Comment by dfreeman** [comment]
Fixed in #637
